mne/data/coil_def_Elekta.dat

```diff
@@ -39,6 +39,12 @@
 #			2	accurate
 #
 #
+1   2000    2   1  0.000e+00  0.000e+00	"Point magnetometer, x-normal"
```

Comment:
Should I do that by also modifying `coil_def.dat`? I can make 2000, 2001, 2002 match x, y, z that way